src/v/cluster/tm_stm.cc

```diff
@@ -684,7 +684,7 @@ ss::future<tm_stm::op_status> tm_stm::add_group(
     if (tx.status != tm_transaction::tx_status::ongoing) {
         vlog(
           txlog.warn,
-          "Expected an ongoing txn, found tx:{} tx:{} pid:{} tx_seq:{} etag:{} "
+          "Expected an ongoing txn, found tx:{} pid:{} tx_seq:{} etag:{} "
```

Comment:
tm_transaction has an << overload that has a consistent formatting, switch to that?
(feel free to skip if there too many places, we can do it as a separate change).

Reply:
oh that's convenient. yeh i think those should be fixed in bulk. this fix is necessary now because the mismatch in number of {} and args causes the entire log line to not print out anything useful.